README.md

```diff
@@ -0,0 +1,126 @@
+# General registry
+
+The official registry of general Julia packages. This registry of packages is
+what connects the `Pkg` standard library (see
+<https://julialang.github.io/Pkg.jl/v1/getting-started/>) to the large ecosystem
+of Julia packages, allowing them to be installed and updated easily.
```

Comment:
I would maybe describe what a registry is rather than paint it as General being "unique".

Reply:
I just copied the tagline from the repo, but I'm open to changing the wording here-- any suggestions?